public/components/common/query_utils/index.ts

```diff
@@ -5,8 +5,8 @@
 
 import dateMath from '@elastic/datemath';
 import { Moment } from 'moment-timezone';
-import { isEmpty } from 'lodash';
-import { SearchMetaData } from '../../event_analytics/redux/slices/search_meta_data_slice';
+import _, { isEmpty } from 'lodash';
+import { SearchMetaData } from 'public/components/event_analytics/redux/slices/search_meta_data_slice';
```

Comment:
should this be a relative import?

Reply:
not sure how this changed, reverted it back